### ISSUE TITLE
Walk through of Issues

### ISSUE THREAD

**Issue by GrassRootsCrypto** [asgardex/asgardex-desktop#241]
## Description

1. Arb.GLD send not working. API Error with key. 
```
{
    "status": "0",
    "message": "NOTOK",
    "result": "Error! Missing Or invalid Module name"
}
```
Issue happens with ARB.WBTC and ARB.ARB also

2. RUNE and Cacao history is in the wrong order. 
Dex: Maya, Rune details page. History in chronological order instead of reverse chronological order. Most recent activity should be at the top, not at the bottom. 
Also happens with Dex = THOR and viewing Cacao. RUNE history doesn't show when Dex = THOR. 
Other assets are fine - just RUNE and Cacao when on asset details. 

3. Swapping Dash to ARB/ETH Fails. 
![image](https://github.com/asgardex/asgardex-desktop/assets/40199525/a5dc4e86-fab9-4920-9bf5-3590c1d27489)

**Comment by GrassRootsCrypto** [comment]
Point 1 is fixed with https://github.com/asgardex/asgardex-desktop/pull/244 and tested.
Waiting on Point 2 and 3. 
Point 3 is fixed by https://github.com/asgardex/asgardex-desktop/pull/250

**Comment by GrassRootsCrypto** [comment]
Parking issue 2 for now. Will create a new issue for this and fix in a future release.
